microcosm_flask/swagger/schemas.py

```diff
@@ -35,6 +37,10 @@ def __init__(
         self.build_parameter = build_parameter
         self.strict_enums = strict_enums
 
+        # XXX This will break if this class is ever instantiated and then has
```

Comment:
I think you can remove the `XXX`

Reply:
@afallou Why remove the `XXX`?  Because we have no intention of ever fixing?  Makes sense I guess.  Maybe `NB`?